File: core/src/org/rexellentgames/dungeon/debug/ArcadeCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.Dungeon;
import org.rexellentgames.dungeon.entity.level.Level;

public class ArcadeCommand extends ConsoleCommand {
	{
		shortName = "/ar";
		name = "/arcade";
		description = "Toggles arcade mode";
	}

	@Override
	public void run(Console console, String[] args) {
		Dungeon.type = Dungeon.Type.ARCADE;
		Dungeon.newGame();
	}
}

File: core/src/org/rexellentgames/dungeon/debug/ArcadeCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.Dungeon

class ArcadeCommand: ConsoleCommand("/arcade", "/ar", "Toggles arcade mode") {
    override fun run(console: Console, args: Array<String>) {
        Dungeon.type = Dungeon.Type.ARCADE
        Dungeon.newGame()
    }
}